source/image_perspective.py

```diff
@@ -0,0 +1,312 @@
+"""Image perspective is a library to turn a photo of a trapezium into a rectangle.
+
+*Use the `warp_perspective()` function to get started...*
+
+Works well with .jpg for now
+
+## BUG & TODO
+---
+- TODO test `get_whiteboard_dimensions` for weird angles
+- BUG `get_whiteboard_dimensions` gives the wrong dimensions
+- BUG `get_coordinates_transform_points` works different for .png .bmp. ***(JPG is tested)***
+- TODO `get_coordinates_transform_points` add support for `cv2`'s camera frames"""
+
+import common.location
+import cv2
+import numpy
+import os
+import time
+
+# Based upon the following works:
+# https://learnopencv.com/automatic-document-scanner-using-opencv/  
+
+
+def resize_image_to_fit_screen(image :cv2.Mat) -> cv2.Mat:
+    """Grabs the resolution of the **Primary** screen and resizes the image to fit both demotions
```

Comment:
explanation of every function is really nice and well detailed :)

Reply:
Thank you @Maucee 😄  !